packages/fluentui/docs/src/components/MarkdownPage.tsx

```diff
@@ -19,27 +19,45 @@ type MarkdownPageProps = {
   };
 } & RouteProps;
 
-const components = {
+const components = teamsTheme => ({
```

Comment:
```suggestion
const components = {
```

This change can be restored

Reply:
done